pages/insight_vacancies.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+// Gives the page a title
+setTitle("Vacancies");
+// Fetching all data that is required for vacancies
+$query = base_query('SELECT * FROM Vacancy')->fetchAll();
+?>
+<p style="font:16px arial, sans-serif;width:75%;">Wij zijn een leuk en gezellig team en op zoek naar jou. En wie zegt er nou nee tegen heerlijk Indonesisch eten. Dan ben je bij ons op het juiste adres.
```

Comment:
Style is handiger om in een aparte style tag te doen. We kunnen het dan makkelijk naar een ander bestand doen later.